README.md

```diff
@@ -65,24 +65,30 @@ Optional pattern to selectively select it/describe cases to run from spec files.
 Type: `RegExp | string`<br>
 Default: undefined
 
-## invertGrep
+### invertGrep
 Inverts 'grep' matches
 
 Type: `Boolean`<br>
 Default: false
 
-## cleanStack
+### cleanStack
 Cleans up stack trace and removes all traces of node module packages
 
 Type: `Boolean`<br>
 Default: true  
 
-## random  
+### random  
 Run specs in semi-random order  
 
 Type: `Boolean`<br>
 Default: `false`
 
+### stopOnSpecFailure
+Stops spec execution on first fail (other specs continue running). Requires Jasmine 3.2+
```

Comment:
We can remove
```
Requires Jasmine 3.2+
```
since this project will always run with Jasmine 3.2+

Reply:
done

Reply:
https://github.com/webdriverio/webdriverio/pull/3086 build failed, they use Jasmine 3.1